### ISSUE TITLE
Export when changing date format

### ISSUE THREAD

**Issue by bepolymathe** [haydenull/logseq-plugin-agenda#7]
Hi @haydenull 

I’am trying to organize my full calendar with french date format. 

![image](https://user-images.githubusercontent.com/29939805/157205176-de02ce0a-f2ca-450f-a449-b43d4dfbf570.png)

But when i try to export, logs are empty 

![image](https://user-images.githubusercontent.com/29939805/157205295-faee5b02-9d54-4ab9-a478-224b3ff64dc7.png)

**Comment by haydenull** [comment]
Weekly Logs Modal shows the content under the `Log Key` Config in journal note, regardless of `journal date formatter` config, is there any content under your journal note's [[Journal]] block?

You could use query to test: `{{query [[Journal]] }}`
<img width="869" alt="image" src="https://user-images.githubusercontent.com/17695989/157214860-4da4abd2-1167-4c25-a7b6-e668991e0480.png">

**Comment by bepolymathe** [comment]
Yes i have content under [[Journal]]

```
- [[Journal]]
 - Event for all day time
 - 10:00-12:00 Meeting with Bob
 - 14:00 Call Alice 
```

**Comment by haydenull** [comment]
Are these block in your journal notes?

**Comment by bepolymathe** [comment]
> Are these block in your journal notes?

Yes

**Comment by haydenull** [comment]
Will they appear in the calendar?

**Comment by haydenull** [comment]
Will they appear in the calendar?

**Comment by bepolymathe** [comment]
![image](https://user-images.githubusercontent.com/29939805/157218084-28e1df02-9316-49c2-a79b-f8df27a6b7de.png)

![image](https://user-images.githubusercontent.com/29939805/157218196-d86c6b51-331e-4186-bd4a-a6c16a986584.png)

![image](https://user-images.githubusercontent.com/29939805/157218254-8d914e25-c144-449c-914e-961d71ddc594.png)

**Comment by haydenull** [comment]
Set journal date formatter to `DD-MM-YYYY`?

**Comment by bepolymathe** [comment]
I try it but nothing change :thinking:

**Comment by haydenull** [comment]
An error occurred while getting the plugin configuration. It has been fixed in version 1.1.2. You can try again.

**Comment by bepolymathe** [comment]
Thanks @haydenull 
Confirm it works !
